### PR TITLE
ensure that multi-valued fields on show page are rendered as links

### PR DESCRIPTION
**Pull request by quoideneuf**

### CODE REVIEW

File: app/views/catalog/_show_fields.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <% else %>
   <dd class="blacklight-<%= field_name.parameterize %> col-xs-12 col-sm-9">
     <ul>
-    <% field_presenter.values.each do |value| %>
+    <% Array.wrap(field_presenter.render).each do |value| %>
```
Comment (dkinzer) [inline app/views/catalog/_show_fields.html.erb:8]: @nomadicoder won't this basically always render something even where there are no fields to render?  `Array.wrap(nil)` will render stuff.

Comment (quoideneuf) [reply app/views/catalog/_show_fields.html.erb]: @dkinzer - good catch, added some tests and reworked the template a bit.

**Comment by quoideneuf** [comment]
this needs more work - I'll reopen shortly

**Comment by quoideneuf** [comment]
I can't seem to reopen this...

**Pull request closed without merging**